lib/touch.js

```diff
@@ -78,8 +85,18 @@ function addPressEvent(target, event, options) {
         return handleTouch();
     };
 
+
+    let timeOut;
     target.onTouchEnded = (isInside) => {
-        endTouch(isInside);
+        timeOut !== undefined && clearTimeout(timeOut);
```

Comment:
```suggestion
        clearTimeout(timeOut);
```
No need to check: https://github.com/smartface/sf-extension-utils/blob/rippleEffectTouch/lib/base/timers.js#L40